client/flags.go

```diff
@@ -27,6 +27,7 @@ func GetCommands(cmds ...*cobra.Command) []*cobra.Command {
 		c.Flags().String(FlagChainID, "", "Chain ID of tendermint node")
 		c.Flags().String(FlagNode, "tcp://localhost:46657", "<host>:<port> to tendermint rpc interface for this chain")
 		c.Flags().Int64(FlagHeight, 0, "block height to query, omit to get most recent provable block")
+		c.Flags().Int64(FlagGas, 200000, "gas limit to set per-transaction")
```

Comment:
This line should be deleted, `FlagGas` should be in PostCommands - https://github.com/cosmos/cosmos-sdk/pull/1182/files#diff-38b4a4a9824366646939d96f3dc02184R41